hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/s3guard/ITestS3GuardToolLocal.java

```diff
@@ -165,18 +165,19 @@ public void testImportNoFilesystem() throws Throwable {
 
   @Test
   public void testInfoBucketAndRegionNoFS() throws Throwable {
-    intercept(FileNotFoundException.class,
+    intercept(UnknownStoreException.class,
         () -> run(BucketInfo.NAME, "-meta",
             LOCAL_METADATA, "-region",
             "any-region", S3A_THIS_BUCKET_DOES_NOT_EXIST));
   }
 
   @Test
   public void testInitNegativeRead() throws Throwable {
-    runToFailure(INVALID_ARGUMENT,
-        Init.NAME, "-meta", LOCAL_METADATA, "-region",
-        "eu-west-1",
-        READ_FLAG, "-10");
+    intercept(CommandFormat.UnknownOptionException.class,
```

Comment:
if this is triggering bucket probe failures, it's not doing the -ve arg check. So here we should either switch to a probe option of 0, or just cut the test entirely. I'm almost going for the latter given the way people should be creating tables is with read = write =0 for on-demand use

Reply:
I agree to remove the test. The error caused by treating `-10` as option rather than option's value since it `startsWith("-")`. The read flag does not need to handle negative value.